### PR TITLE
Runv subcommand

### PR DESCRIPTION
**Pull request by laijs**
as runc's subcommand

### CODE REVIEW

**Comment by gao-feng** [comment]
why introduce the subcommands ? is there other subcommands except start?

**Comment by laijs** [comment]
Other subcommands will be added.
The goal is to make runv more compatible with the runc as we can.
The user don't need to learn to much when he/her knows runc.

**Comment by gao-feng** [comment]
compatible with runc is good, with this patch, can `runv` successfully run without start subcommand?

**Comment by laijs** [comment]
Yes, it can, the same as runc.
The older command line still work

File: main.go [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    "linux" and "mount" fields in OCI specs are ignored
+
+The current release of "runV" supports the following hypervisors:
+    KVM (QEMU 2.0 or later)
+    Xen (4.5 or later)
+    VirtualBox (Mac OS X)
+
+After creating a spec for your root filesystem, you can execute a container
+in your shell by running:
+
```
Comment (gao-feng) [inline main.go]: The above messages looks wired stay in usage...

File: main.go [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+			Usage: "[WIP] enable debug output for logging",
+		},
+		cli.StringFlag{
+			Name:  "log",
+			Usage: "[WIP] set the log file path where internal debug information is written",
+		},
+		cli.StringFlag{
+			Name:  "root",
+			Value: "/run/oci",
+			Usage: "[WIP] root directory for storage of container state (this should be located in tmpfs)",
```
Comment (gao-feng) [inline main.go]: Add unsupport tag in the usage of these  four flags?

**Comment by gao-feng** [comment]
had better to add some commit logs to show how to use runv.

File: main.go (hunk 1 above)
Comment (laijs) [inline main.go]: The first line reads it supports OCF, but actually it is not fully support, so the compatible note is required after it.
The hypervisor section is also useful information here.

**Comment by laijs** [comment]
`runv --help` will shows how to use runv

File: main.go (hunk 2 above)
Comment (laijs) [inline main.go]: They will be supported.

File: main.go (hunk 1 above)
Comment (gao-feng) [inline main.go]: these messages are already on runv readme. usage is used to show how to use this program, isn't it?

File: main.go (hunk 2 above)
Comment (gao-feng) [inline main.go]: Then add them back with the implementation?

**Comment by laijs** [comment]
updated, merge it

**Pull request merged**